install_config/configuring_authentication.adoc

```diff
@@ -55,9 +59,47 @@ configuration file.
 xref:../architecture/additional_concepts/authorization.adoc#roles[Roles] need
 to be assigned to administer the setup with an external user.
 ====
+ 
+[[identity-providers-ansible]]
+== Configuring Identity Providers with Ansible
 
-[[identity-providers]]
+For initial xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced
+installations], the
+xref:../install_config/configuring_authentication.adoc#DenyAllPasswordIdentityProvider[Deny All] identity provider is configured by default,
+though it can be
+xref:../install_config/install/advanced_install.adoc#configuring-ansible[overridden during installation] using the
+xref:../install_config/install/advanced_install.adoc#configuring-cluster-variables[`*openshift_master_identity_providers*` parameter], which is configurable in the inventory file.
+
+.Example Identity Provider Configuration with Ansible
```

Comment:
Do we need links / examples for https://docs.openshift.com/enterprise/3.2/install_config/install/advanced_install.html#advanced-install-session-options